installer/home/pom.xml

```diff
@@ -75,6 +75,7 @@
                                     <goal>copy-resources</goal>
                                 </goals>
                                 <configuration>
+                                	<encoding>UTF-8</encoding>
```

Comment:
I think this is redundant with the project.build.sourceEncoding property that is in pull request #64.

Reply:
@grahamtriggs : It looks like you are trying to reference a different PR. https://github.com/vivo-project/Vitro/pull/64 is "Update license year".

Reply:
@awoods I think @grahamtriggs wanted to refer #68 where he set the project.build.sourceEncoding property.

That being said I think this PR on Vitro is just useless as the rest of commits where all reverted. Would you please delete it ?